src/en/space-station-14/mapping/guides/general-guide.md

```diff
@@ -62,18 +64,18 @@ Now to start creating maps follow the below steps:
 2. Once the game client loads up, connect to the server.
 3. Press the backtick/tilde key <kbd>`</kbd> to open the debug console, which allows you to run console commands.
 4. Run the `mapping [MapID] [MapFile]` command to either create a new map or edit an existing one.
-    - To make a new map, specify only the map-id. The ID must be free, i.e., the server cannot currently have another map loaded with the same ID).
-    - To edit an existing map in the Maps/ folder you need to specify the map file name. For example, in order to edit the saltern map you can use something like `mapping 2 Maps/saltern.yml`
-    - To edit a map you have previously saved you can run the command `mapping 2 [filename.yml]`. These are found in the bin/content.server/data folder.
+    - To make a new map, specify only the map-id. The ID must be free, i.e., the server cannot currently have another map loaded with the same ID), as such you should use a sufficiently high ID to avoid a conflict.
+    - To edit an existing map in the Maps/ folder you need to specify the map file name and a free ID. For example, in order to edit the Saltern map you can use something like `mapping 1000 Maps/saltern.yml`
+    - To edit a map you have previously saved you can run the command `mapping 1000 [filename.yml]`. Your saved maps are found in the `bin/content.server/data` folder, separate from the maps used by the repository in the `Resources/Maps` folder.
```

Comment:
Could you fix the case of the Content.Server folder? On some systems, case matters.

Reply:
Case fixed